### PR TITLE
kafka-4866: Kafka console consumer property is ignored

### PR DESCRIPTION
**Pull request by huxihx**
Added `print.value` config in ConsoleConsumer to match what the quickstart document says.

### CODE REVIEW

**Comment by huxihx** [comment]
@ijuma a minor improvement. Please review. Thanks.

**Comment by ijuma** [comment]
Thanks for the PR, can we add a test please?

**Comment by huxihx** [comment]
@ijuma sorry for the delay. Added a test item in `shouldParseValidNewSimpleConsumerValidConfigWithStringOffset` method for explicitly setting `print.value` to false. Could you review again? Thanks.

**Comment by huxihx** [comment]
@ijuma Do you have any chances to review the PR?

**Comment by huxihx** [comment]
@hachikuji Could you help review this PR? Thanks

File: core/src/main/scala/kafka/tools/ConsoleConsumer.scala [hunk 1]
```diff
@@ -455,7 +458,11 @@ class DefaultMessageFormatter extends MessageFormatter {
     }
 
     if (printKey) write(keyDeserializer, key, keySeparator)
-    write(valueDeserializer, value, lineSeparator)
+    if (printValue) {
+      write(valueDeserializer, value, lineSeparator)
+    } else {
+      write(None, "".getBytes(), lineSeparator)
```
Comment (ijuma) [inline core/src/main/scala/kafka/tools/ConsoleConsumer.scala]: Isn't it better to do `output.write(lineSeparator)` instead?

**Comment by huxihx** [comment]
@ijuma Sounds good. Already addressed your comments. Please review again.

File: core/src/main/scala/kafka/tools/ConsoleConsumer.scala [hunk 2]
```diff
@@ -455,7 +458,11 @@ class DefaultMessageFormatter extends MessageFormatter {
     }
 
     if (printKey) write(keyDeserializer, key, keySeparator)
```
Comment (ijuma) [inline core/src/main/scala/kafka/tools/ConsoleConsumer.scala]: Maybe we should not output `keySeparator` is we are not printing values?

**Comment by huxihx** [comment]
@ijuma Added a helper function `outputSeparator` to cover all the 7 situations where `printTimestamp`, `printKey` and `printValue` are set with different values.

**Comment by huxihx** [comment]
@ijuma Did you get any chance to review the new merged code? Thanks in advance.

**Comment by ijuma** [comment]
Thanks for the PR, LGTM. Merged to trunk after some minor tweaks.

**Pull request closed without merging**